README.md

```diff
@@ -2,16 +2,76 @@
 
 [![License](https://img.shields.io/badge/License-Apache-blue.svg)](LICENSE)
 
-## Description
+# Realm Dart Samples
 
-This repo contains Realm Flutter and Realm Dart SDK samples. 
+This repo contains Flutter and Dart samples that demonstate the usage of [Realm Flutter and Dart Standalone SDKs](https://www.mongodb.com/docs/realm/sdk/flutter/).
 
-The Realm Flutter and Realm Dart SDK are located at https://github.com/realm/realm-dart
+To build your own app with Realm, use the [realm](https://pub.dev/packages/realm) package for Flutter and the [realm_dart](https://pub.dev/packages/realm_dart) package for Dart applications.
 
+The Realm Flutter and Realm Dart SDK repository is located at https://github.com/realm/realm-dart
 
-## Usage 
-Check the README.md files of the samples for instructions how to run them.
+You may find the [Quick Start - Flutter SDK]( https://www.mongodb.com/docs/realm/sdk/flutter/quick-start/) helpful if you are using Realm for the first time.
+To learn more about using Realm with Atlas App Services, refer to the [Atlas Device Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) and [connecting to Atlas App Services](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/) documentation.
 
+## [`realm_dart`](https://github.com/realm/realm-dart-samples/tree/main/realm_dart) - Dart console application
 
+This sample uses the [Realm Dart Standalone SDK](https://www.mongodb.com/docs/realm/sdk/flutter/#dart-standalone-realm) to demonstrates the following simple operations with a local realm:
 
-##### The "Dart" name and logo and the "Flutter" name and logo are trademarks owned by Google. 
+- Create and open local realm
```

Comment:
I suggest we list all samples with single sentence about what it is. then in each sample we can have all the details about it. So removing this whole section below

Reply:
Done.